docs/toc.yml

```diff
@@ -664,8 +667,9 @@
     - name: Versioning
       href: csharp/versioning.md
   - name: How To C# Topics
-    href: csharp/how-to/index.md
     items:
+    - name: Topic index
```

Comment:
Need to think about this one...

Reply:
I agree. this one is another short-term work around to make it a bit better before doing the long term fix.

The long term fix as part of the content model updates (see #11606) is to move the "How to" articles further down-level in the TOC: The "strings" how to articles with conceptual content on strings, how to LINQ content with LINQ content and so on. This page would go away under that format. Until then, it's useful as a roadmap to all the "how to" content distributed throughout the guide.